File: IMUOJ/1308.cpp
#include<iostream>
using namespace std;
int f(int);
int main()
{
	int x;
	while (cin >> x)
	{
		if (x == 0)break;
		cout << f(x) << endl;
	}
}
int f(int x)
{
	if (x == 1)return 1;
	else if (x == 2) return 2;
	else if (x == 3)return 4;
	else return f(x - 1) + f(x - 2) + f(x - 3);
}

File: LUOGU/1308.java
import java.util.Scanner;

/**
 * Created by likole on 10/22/18.
 */
public class Main {

    public static void main(String[] args) {
        Scanner cin = new Scanner(System.in);
        String target, s;
        target = cin.nextLine();
        s = cin.nextLine();
        target = target.toLowerCase();
        s = s.toLowerCase();
        target=" "+target+" ";
        s=" "+s+" ";
        if(!s.contains(target)){
            System.out.println(-1);
        }else{
            int cnt=1;
            int pos=s.indexOf(target);
            int first=pos;
            while ((pos=s.indexOf(target,pos+1))!=-1){
                cnt++;
            }
            System.out.println(cnt+" "+first);
        }

    }
}

